server/tests/message.integration.spec.js

```diff
@@ -311,49 +392,104 @@ describe('Message API:', function () {
     });
 
     // TODO: this one is going to be hard
-    describe('GET /message/thread/:originalMessageId', () => {
-        
+    // describe('GET /message/thread/:originalMessageId', () => {
+
+    //     let messageId;
+
+    //     before(done => {
+    //         Message.create(testMessageObject)
+    //             .then(message => {
+    //                 originalMessageId = message.originalMessageId;
+    //                 done();
+    //             });
+    //     });
+
+    //     // TODO: create a real response message here
+
+    //     it('should return OK', done => {
+    //         request(app)
+    //             .get(baseURL + '/message/thread' + originalMessageId)
+    //             .expect(httpStatus.OK)
+    //             .then(res => {
+    //                 expect(res.text).to.equal('OK');
+    //                 done();
+    //             })
+    //             .catch(done);
+    //     });
+
+    //     it('should return an array of message IDs, starting with the original message', done => {
+    //         request(app)
+    //             .get(baseURL + '/message/thread' + originalMessageId)
+    //             .expect(httpStatus.OK)
+    //             .then(res => {
+    //                 expect(res.body).to.be.an.array;
+    //                 // TODO check specific IDs
+    //                 done();
+    //             })
+    //             .catch(done);
+    //     });
+
+    // });
+
+    describe('DELETE /message/delete/:messageId', function () {
+
         let messageId;
-        
-        before(done => {
-            Message.create(testMessageObject)
-                .then(message => {
-                    originalMessageId = message.originalMessageId;
-                    done();
-                });
+
+        beforeEach(done => {
+            Message.destroy({
+                where: {},
+                truncate: true
+            }).then(() => {
+                Message.create(testMessageObject)
+                    .then(message => {
+                        messageId = message.id;
+                        done();
+                    });
+            });
         });
 
-        // TODO: create a real response message here
+        it('should return OK', () => {
+            request(app)
+                .get(baseURL + '/message/delete/' + messageId)
+                .set('Authorization', `Bearer ${auth}`)
+                .expect(httpStatus.OK)
+        });
 
-        it('should return OK', done => {
+        it('should return the deleted Message', done => {
             request(app)
-                .get(baseURL + '/message/thread' + originalMessageId)
+                .delete(baseURL + '/message/delete/' + messageId)
+                .set('Authorization', `Bearer ${auth}`)
                 .expect(httpStatus.OK)

```

Comment:
I think we were doing to change the HTTP status here to `204`, or at least something other than `OK` for deletions.